settings.gradle

```diff
@@ -0,0 +1,65 @@
+pluginManagement {
+    repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+rootProject.name = "NDK Samples"
+include(":audio-echo:app")
+include(":bitmap-plasma:app")
+include(":camera:basic")
+include(":camera:texture-view")
+include(":display-p3:image-view")
+include(":endless-tunnel:app")
+include(":exceptions:app")
+include(":gles3jni:app")
+include(":hello-gl2:app")
+include(":hello-jni:app")
+include(":hello-jniCallback:app")
+include(":hello-libs:app")
+// To generate libs used in this sample:
+//   1) enable the gen-libs at end of this file
+//   2) enable module build dependency in app/build.gradle
+//   3) build the app's APK in Android Studio or on command line
+//   4) undo step 1) and 2) above
+// include(":hello-libs:gen-libs")
+include(":hello-neon:app")
+include(":hello-oboe:app")
+//include(":hello-vulkan:app")
```

Comment:
Oops, I forgot to come back to this.

Reply:
https://github.com/android/ndk-samples/pull/1027 fixes the problem with the sample.